src/dos/program_imgmount.cpp

```diff
@@ -36,12 +36,56 @@
 #include "string_utils.h"
 #include "../ints/int10.h"
 
+void IMGMOUNT::ListImgMounts(void)
+{
+	const std::string header_drive = MSG_Get("PROGRAM_MOUNT_STATUS_DRIVE");
+	const std::string header_name = MSG_Get("PROGRAM_MOUNT_STATUS_NAME");
+	const std::string header_label = MSG_Get("PROGRAM_MOUNT_STATUS_LABEL");
+	const std::string header_slot = MSG_Get("PROGRAM_MOUNT_STATUS_SLOT");
+
+	const int term_width = real_readw(BIOSMEM_SEG, BIOSMEM_NB_COLS);
+	const auto width_1 = static_cast<int>(header_drive.size());
+	const auto width_3 = std::max(11, static_cast<int>(header_label.size()));
+	const auto width_4 = std::max(11, static_cast<int>(header_slot.size()));
+	const auto width_2 = term_width - 4 - width_1 - width_3 - width_4;
+
```

Comment:
If some of these widths are coming in from the messaging system, then they might be written by translators, and therefore are truly "user-side content".

My concerns is that, if some of those messages are quite wide, the subtractions could take width_2 into negative territory. 

I'm not sure if there's a hard-minimum or if width_2 is allowed to go negative? (maybe it's OK?), but if not, a runtime check could help.

Reply:
I added a run-time check as suggested. Done.